src/components/Layout/Header/NavBar/MainNavLink.tsx

```diff
@@ -24,68 +24,72 @@ const activeProp = {
   fontWeight: 'bold',
 }
 
-export const MainNavLink = memo(
-  ({ isCompact, onClick, isNew, label, isActive, ...rest }: SidebarLinkProps) => {
-    const [isLargerThan2xl] = useMediaQuery(`(min-width: ${breakpoints['2xl']})`, { ssr: false })
-    const translate = useTranslate()
-    const handleClick: React.MouseEventHandler<HTMLButtonElement> = useCallback(
-      e => (isActive ? e.preventDefault() : onClick?.(e)),
-      [isActive, onClick],
-    )
+export const MainNavLink = memo((props: SidebarLinkProps) => {
+  const { isCompact, onClick, isNew, label, isActive, buttonProps } = useMemo(() => {
+    const { isCompact, onClick, isNew, label, isActive, ...buttonProps } = props
+    return { isCompact, onClick, isNew, label, isActive, buttonProps }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, Object.values(props))
```

Comment:
Same q. re: all the `Object.values()` usages in this PR, this is probably not the reactivity we want

Reply:
as discussed, keeping this as it seems to be suitable